setup.py

```diff
@@ -77,6 +77,7 @@
 
 install_requires = [
     'Click>=6.3',
+    'Flask-Security>=1.7.5',
```

Comment:
This should be added in ``extras_require`` under either ``loader`` or ``userprofiles``. This is due to the package being installed on both legacy and invenio 3 systems.